src/wp-includes/comment.php

```diff
@@ -3297,7 +3297,6 @@ function update_comment_cache( $comments, $update_meta_cache = true ) {
  * Adds any comments from the given IDs to the cache that do not already exist in cache.
  *
  * @since 4.4.0
- * @access private
```

Comment:
Suggest adding `@since 6.1.0 This function is no longer marked as "private".` to all of these changes.

See [changeset 46602](https://core.trac.wordpress.org/changeset/46602) for precedent.